### PR TITLE
Addapted files, so docs are accessible in docs.debops.org

### PR DESCRIPTION
**Pull request by AnBuKu**

### CODE REVIEW

**Comment by ypid** [comment]
Thanks @AnBuKu 

Looking into this PR and reworking it at bit as kindly requested by @AnBuKu 😉

@AnBuKu For proper documentation, I guess more technical details will need to be added to docs over time. I will do the basics and get metadata and common docs up to speed. After that, I can open a PR against your work so that you can include my improvements into this PR.

**Comment by drybjed** [comment]
@ypid Don't worry about documenting this role extensively just yet. I want to move parts of it out to separate roles over time. Cramming lots of different things into one role was a mistake on my part.

**Comment by ypid** [comment]
All right. That was also my impression that the role was scheduled for rework.

**Comment by ypid** [comment]
Opened an update for this PR: https://github.com/AnBuKu/ansible-auth/pull/1

**Comment by ypid** [comment]
@AnBuKu Can you merge https://github.com/AnBuKu/ansible-auth/pull/1 ?

File: docs/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Getting started
+===============
+
+.. contents::
+   :local:
+
+Initial configuration
+---------------------
+
+``debops.auth`` will instal
```
Comment (ypid) [inline docs/getting-started.rst]: You can use https://raw.githubusercontent.com/debops/ansible-elasticsearch/master/docs/getting-started.rst as template.

Comment (AnBuKu) [inline docs/getting-started.rst]: Thx, I'll try week comming

File: defaults/main.yml [hunk 2]
```diff
@@ -146,7 +146,7 @@ auth_ldap_conf_options: |
   TIMELIMIT        15
   DEREF            never
 
-# ]]]
+                                                                   # ]]]
```
Comment (ypid) [inline defaults/main.yml]: Have a little trust in [yaml4rst](https://yaml4rst.readthedocs.io/en/latest/) 😉 It follows all the best practices and is heavily unit and integration tested :sunglasses: (Background: `auth_ldap_conf_options` is a YAML text block which includes everything indented more or the same as the first line of the block. We don’t want the folding comment to end up in config files)

Comment (AnBuKu) [inline defaults/main.yml]: Oh, I trust in every code you do. :-)

I was simply not expecting, my little change in https://github.com/AnBuKu/ansible-auth/blob/master/defaults/main.yml will have consequences here...

Comment (ypid) [inline defaults/main.yml]: > Oh, I trust in every code you do. :-)

Thanks :)

> my little change in https://github.com/AnBuKu/ansible-auth/blob/master/defaults/main.yml will have consequences here.

Ah, it is all the same branch (`master`) which keeps it simple. I guess you can change it back as it was before (indented to 78 characters).

**Pull request merged**